test/helpers/lsp.cc

```diff
@@ -347,4 +347,54 @@ unique_ptr<LSPMessage> makeChange(std::string_view uri, std::string_view content
     return make_unique<LSPMessage>(move(didChangeNotif));
 }
 
+vector<unique_ptr<LSPMessage>> getLSPResponsesFor(LSPWrapper &wrapper, vector<unique_ptr<LSPMessage>> messages) {
```

Comment:
Tests use this method so we can share logic that operates in both single and multithreaded mode.